<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>val3PrecompilerSettings</class>
 <widget class="QWidget" name="val3PrecompilerSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>362</width>
    <height>168</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>VAL3 Precompiler Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QMenuBar" name="menuBar"/>
   </item>
   <item>
    <widget class="QCheckBox" name="cpUnusedPublicGlobalVar">
     <property name="text">
      <string>Report public, global variables that are not used in the application.</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="cpHiddenGlobalVariables">
     <property name="text">
      <string>Report global variables that are hidden by local variables.</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="cpParametersPostfix">
     <property name="text">
      <string>Report program parameters that do not have postfix &quot;_&quot;.</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="cpReportsToDos">
     <property name="text">
      <string>Report TODOs.</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
